README.md

```diff
@@ -22,6 +22,12 @@ EvalML is an AutoML library which builds, optimizes, and evaluates machine learn
 pip install evalml
 ```
 
+### Add-ons
```

Comment:
Ultra nit-pick: use #### instead of ### to be consistent with the rest of the readme